microservices-orders/docs/index.rst

```diff
@@ -223,11 +223,13 @@ Exercise 0: Run end-to-end example
 
 Running the fully working example end-to-end provides context for each of the later exercises.
 
-#. Ensure you've followed the appropriate prerequisites section above prior to starting.
+Ensure you've followed the appropriate prerequisites section above prior to starting.
 
 #. Start the end-to-end example in one of three modes, depending on whether you are running with |ccloud|, in Docker, or |cp| locally:
 
-   * |ccloud|: first log in to |ccloud| with the command ``ccloud login``, and use your |ccloud| username and password. To prevent being logged out, use the ``--save`` argument which saves your |ccloud| user login credentials or refresh token (in the case of SSO) to the local ``netrc`` file. Then run the full solution using the provided script (this starts a new |ccloud| environment and Kafka using the :ref:`ccloud-stack`).
+   * |ccloud|: first log in to |ccloud| with the command ``ccloud login``, and use your |ccloud| username and password. To prevent being logged out, use the ``--save`` argument which saves your |ccloud| user login credentials or refresh token (in the case of SSO) to the local ``netrc`` file. Then run the full solution using the provided script.
```

Comment:
Since we're using preformatted text on the netrc part, maybe we need to be specific?

```suggestion
   * |ccloud|: first log in to |ccloud| with the command ``ccloud login``, and use your |ccloud| username and password. To prevent being logged out, use the ``--save`` argument which saves your |ccloud| user login credentials or refresh token (in the case of SSO) to your home ``.netrc`` file. Then run the full solution using the provided script.
```